Luke Smith Portfolio.css

```diff
@@ -0,0 +1,111 @@
+/** {
+	box-sizing: border-box;
+}
+*/
```

Comment:
Try not to leave commented-out code in your files. If this is for further testing then leave it for now, but make sure it's deleted before you send it off for final review